src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -373,6 +422,7 @@ def _save_privacy_preferences_for_identities(
                 if verified_provided_identity
                 else None,
                 "request_origin": request_data.request_origin,
+                "served_notice_history_id": privacy_preference.served_notice_history_id,
```

Comment:
This gives the FE the option to pass in the `served_notice_history_id` record so we can link together which served notices were converted to saved preferences for that notice.  Note that both "notices served" and "preferences saved" as stored at the _notice_ level but you can pass in multiple notices in both requests to capture everything that was served or saved at the same time.